sdc/src/util/json.d

```diff
@@ -1883,6 +1893,15 @@ unittest {
     toJSON(JSON());
 }
 
+//is order independent for non-nested objects
+version(unittest) auto objectStringTest(R, Rs...)(R res, Rs components)
+{
+    return only(components)
+        .permutations
+        .map!(p => chain(`{`, p.joiner(`,`), `}`))
+        .canFind!equal(res);
+}
```

Comment:
can't assume stable ordering of object members in JSON, same goes for D's associative arrays, hence the need for this helper function to check that the generated JSON object string is some correct permutation or a set of components of a JSON object string. This is not a general solution, it only works for top-level objects.